src/Basket.js

```diff
@@ -1,16 +1,31 @@
 import React from 'react';
 
-const Basket = () => {
-  return (
-    <section>
-      <p>Shopping Cart</p>
-      <ul>
-        <li>Phone 1 <button>x</button></li>
-        <li>Phone 2 <button>x</button></li>
-        <li>Phone 3 <button>x</button></li>
-      </ul>
-    </section>
-  );
-};
+export default class Basket extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            items:[]
+        }
+    }
+    render() {
+        const {removeFromBasket}= this.props;
+        return (
+            <section>
+                <p>Basket</p>
+                <ul>
+                    {
+                        this.props.items && this.props.items.map((item, index) => (
```

Comment:
As far as you pass `array`checking `this.props.items && ...` is useless. Casting array to boolean always returns true.
Also, `map` of empty array returns an empty array.
```suggestion
                        this.props.items.map((item, index) => (
```

Reply:
![image](https://user-images.githubusercontent.com/29413892/59565557-ee8bac00-905d-11e9-83ec-e536081f5869.png)